Bookish/Controllers/CatalogController.cs

```diff
@@ -114,6 +116,22 @@ public IActionResult BookList()
         }
         return View(bvm);
     }
+    
+    [HttpGet]
+    public IActionResult GetCopyList(int inputId)
+    {
+        CopyViewModel cvm = new CopyViewModel();
+        using (var context = new LibraryContext())
+        {
+            cvm.CatalogEntries = context.Copies.Include(b => b.BorrowInstanceList)
+                .Include(b => b.Book)
+                .Where(b => b.BookID == inputId)
+                .ToList();
+            cvm.Book = context.Copies.FirstOrDefault(b => b.BookID == inputId).Book;
+            cvm.Author = context.Authors.SingleOrDefault(a => a.AuthorID == cvm.Book.AuthorID);

```

Comment:
I think doing something like

```cs
context.Books
  .Include(b => b.Copies)
  .Include(b => b.Author)
  .Single(b => b.Id = inputId)
```

and then pulling out the copies and author would be a bit more entity-frameworky, though it doesn't really matter.

Reply:
Ahh yeah this is a much nicer way than the way I did it in the first place! I didn't know the single method was a thing so that's really helpful :)